test/integration/targets/apt/tasks/apt_deb_depend.yml

```diff
@@ -0,0 +1,38 @@
+- block:
+  - name: Clean up before running tests
+    apt:
+      name:
+        - packageone
+        - packagetwo
+      state: absent
+    ignore_errors: true
```

Comment:
Why are errors being ignored here?

Reply:
Copied from existing cleanup tasks. I can remove it.